### PR TITLE
bump up skktools 1.3.4

### PR DESCRIPTION
**Pull request by kyohsuke**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
This has not test block so I add the test and bump up version.

### CODE REVIEW

File: Formula/skktools.rb [hunk 1]
```diff
@@ -24,4 +24,50 @@ def install
...
+
+    expect_expr = <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      ;; okuri-ari entries.
+      わるs /悪/
+      わるk /悪/
+    EOS
+
+    assert_equal "SKK-JISYO.TEST: 3 candidates", shell_output("#{bin}/skkdic-count SKK-JISYO.TEST").strip
+    assert_equal expect_shuffle, shell_output("cat SKK-JISYO.SHUFFLE | #{bin}/skkdic-sort").strip
+    assert_equal expect_expr, shell_output("#{bin}/skkdic-expr SKK-JISYO.SPLIT1 + SKK-JISYO.SPLIT2 - SKK-JISYO.SPLIT3 | #{bin}/skkdic-sort").strip
```
Comment (DomT4) [inline Formula/skktools.rb]: You can use `pipe_output` for these rather than `shell_output`. We want to wrap to 80 characters as well.

Comment (kyohsuke) [reply Formula/skktools.rb]: Hi @DomT4 

Thank you for review.
I fixed it to wrap to 80 characters.

Is it ok? or I misunderstand it?


Thanks.

File: Formula/skktools.rb [hunk 2]
```diff
@@ -24,4 +24,65 @@ def install
...
+
+    expect_expr = <<~EOS.tap { |s| s.encode("euc-jis-2004") }
+      ;; okuri-ari entries.
+      わるs /悪/
+      わるk /悪/
+    EOS
+
+    # test skkdic-count
+    expect_count = "SKK-JISYO.TEST: 3 candidates\n"
+    actual_count = pipe_output("#{bin}/skkdic-count SKK-JISYO.TEST", nil, 0)
```
Comment (DomT4) [inline Formula/skktools.rb]: You can still use `shell_output()` on this one. Just meant where we were piping things it made sense to use `pipe_output()`. Apologies if my prior wording was confusing!

Comment (kyohsuke) [reply Formula/skktools.rb]: > You can still use shell_output() on this one, I think? Just meant where we were piping things it made sense to use pipe_output().

I fixed it, thank you!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/39e844259f92986b977587fbd6fa0f8b620cacc9. Thank you for your contribution to Homebrew @kyohsuke; we appreciate it! :smiley_cat: 

Thank you for taking the time to add such a thorough test as well 🙇.

**Comment by kyohsuke** [comment]
@DomT4 Thank you for your help!

**Pull request closed without merging**